portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/openidconnect/dependencies/com.liferay.portal.security.sso.openid.connect.internal.configuration.OpenIdConnectProviderConfiguration~Azure.config

```diff
@@ -4,8 +4,8 @@ discoveryEndPointCacheInMillis=L"360000"
 idTokenSigningAlgValues=["RS256"]
 issuerURL=""
 jwksURI=""
-openIdConnectClientId="1afe1ae0-f11a-4eeb-9c75-301a07b36551"
-openIdConnectClientSecret="ZGR8Q~CxFM7VzEZv6nWqaF8FMKOofp0vNYrXOaCX"
+openIdConnectClientId="${openid.provider.azure.client.id}"
+openIdConnectClientSecret="${openid.provider.azure.client.secret}"
```

Comment:
Should be openid.provider.azure.client.password:https://github.com/liferay/liferay-jenkins-ee/blob/master/commands/build.properties#L1097